code.go

```diff
@@ -40,10 +39,11 @@ var SourceCodeTypes = map[string]bool{
 }
 
 // parseCode parses a code address directive.
-func parseCode(addr []byte, file []byte) []byte {
+func (p *parser) parseCode(addr []byte, file []byte) []byte {
 	bytes.TrimSpace(addr)
 
-	textBytes, err := ioutil.ReadFile(string(file))
+	fullname := absname(p.cwd, string(file))
```

Comment:
Sure, although is there a benefit in working with []byte? I suspect the performance benefit is neglible, if there is one.

I don't have anything against it either.